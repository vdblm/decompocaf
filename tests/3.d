int main(){
    Print("input your name:");
    Print(ReadLine());
    Print("ok bruh now input your age : ->\n", ReadInteger(), "good age? answer is ", true);
}
