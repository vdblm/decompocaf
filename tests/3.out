input your name:
Agha Sadegh
ok bruh now input your age : ->
20good age? answer is true
